java/android/build-nnstreamer-android-lib.sh

```diff
@@ -0,0 +1,618 @@
+fi
+
+# Update SNPE option
+if [[ $enable_snpe == "yes" ]]; then
+    sed -i "s|ENABLE_SNPE := false|ENABLE_SNPE := true|" nnstreamer/src/main/jni/Android-nnstreamer-prebuilt.mk
+    sed -i "s|ENABLE_SNPE := false|ENABLE_SNPE := true|" nnstreamer/src/main/jni/Android.mk
+    sed -i "$ a SNPE_EXT_LIBRARY_PATH=src/main/jni/snpe/lib/ext" gradle.properties
+
+    mkdir -p nnstreamer/src/main/jni/snpe/lib/ext/arm64-v8a
+    cp -r $SNPE_DIRECTORY/include nnstreamer/src/main/jni/snpe
+    cp $SNPE_DIRECTORY/lib/aarch64-android-clang6.0/libSNPE.so nnstreamer/src/main/jni/snpe/lib
+
+    # Copy external so files for SNPE
+    cp $SNPE_DIRECTORY/lib/aarch64-android-clang6.0/lib*dsp*.so nnstreamer/src/main/jni/snpe/lib/ext/arm64-v8a
+    cp $SNPE_DIRECTORY/lib/aarch64-android-clang6.0/libhta.so nnstreamer/src/main/jni/snpe/lib/ext/arm64-v8a
+    cp $SNPE_DIRECTORY/lib/dsp/libsnpe*.so nnstreamer/src/main/jni/snpe/lib/ext/arm64-v8a
+fi
+
+# Update PyTorch option
+if [[ $enable_pytorch == "yes" ]]; then
+    sed -i "s|ENABLE_PYTORCH := false|ENABLE_PYTORCH := true|" nnstreamer/src/main/jni/Android.mk
+    sed -i "s|PYTORCH_VERSION := 1.8.0|PYTORCH_VERSION := $pytorch_ver|" nnstreamer/src/main/jni/Android-pytorch.mk
+    tar -xJf ./external/pytorch-$pytorch_ver.tar.xz -C ./nnstreamer/src/main/jni
+fi
+
+# Update tf-lite option
+if [[ $enable_tflite == "yes" ]]; then
+    sed -i "s|ENABLE_TF_LITE := false|ENABLE_TF_LITE := true|" nnstreamer/src/main/jni/Android-nnstreamer-prebuilt.mk
+    sed -i "s|ENABLE_TF_LITE := false|ENABLE_TF_LITE := true|" nnstreamer/src/main/jni/Android.mk
+    sed -i "s|TFLITE_VERSION := 1.13.1|TFLITE_VERSION := $tf_lite_ver|" nnstreamer/src/main/jni/Android-tensorflow-lite.mk
+    tar -xJf ./external/tensorflow-lite-$tf_lite_ver.tar.xz -C ./nnstreamer/src/main/jni
+fi
+
+
+if [[ $enable_decoder_flatbuf == "yes" ]]; then
+    sed -i "s|ENABLE_DECODER_FLATBUF := false|ENABLE_DECODER_FLATBUF := true|" nnstreamer/src/main/jni/Android.mk
+    sed -i "s|FLATBUF_VER := @FLATBUF_VER@|FLATBUF_VER := ${decoder_flatbuf_ver}|" nnstreamer/src/main/jni/Android-dec-flatbuf.mk
+    wget --directory-prefix=./external https://raw.githubusercontent.com/nnstreamer/nnstreamer-android-resource/master/external/flatbuffers-${decoder_flatbuf_ver}.tar.xz
+    tar -xJf ./external/flatbuffers-${decoder_flatbuf_ver}.tar.xz -C ./nnstreamer/src/main/jni
+fi
+
+# Add dependency for release
+if [[ $release_bintray == "yes" ]]; then
+    sed -i "s|// add dependency (bintray)|classpath 'com.novoda:bintray-release:0.9.1'|" build.gradle
+
+    sed -i "s|// add plugin (bintray)|apply plugin: 'com.novoda.bintray-release'\n\
+\n\
+publish {\n\
+    userOrg = 'nnsuite'\n\
+    repoName = 'nnstreamer'\n\
+    groupId = 'org.nnsuite'\n\
+    artifactId = '$nnstreamer_lib_name'\n\
+    publishVersion = '$release_version'\n\
+    desc = 'NNStreamer API for Android'\n\
+    website = 'https://github.com/nnstreamer/api'\n\
```

Comment:
line 503 ~ 505 
let's use nnstreamer repo, not api.
https://github.com/nnstreamer/api > https://github.com/nnstreamer/nnstreamer